mix.exs

```diff
@@ -25,16 +30,17 @@ defmodule Explorer.MixProject do
 
   defp deps do
     [
-      {:ex_doc, "~> 0.24", only: :dev, runtime: false},
+      {:ex_doc, "~> 0.24", only: :docs, runtime: false},
       {:nx, "~> 0.1.0"},
-      {:rustler_precompiled, "~> 0.2"},
+      {:rustler_precompiled, "~> 0.3"},
       {:table_rex, "~> 3.1.1"}
     ]
   end
 
   defp docs do
     [
       main: "Explorer",
+      logo: "numbat.png",
```

Comment:
We already have explorer.png, we can't reuse it?

Also make sure to `mix deps.update ex_doc` before. :D

Reply:
We can! The numbat-only one looks a bit better in the top left IMO. I couldn't find where else the logo gets shown.

Reply:
For comparison:

## Numbat only
<img width="361" alt="image" src="https://user-images.githubusercontent.com/3984794/165246828-8205b733-6a15-4363-8134-340fe4d6bbac.png">

## explorer.png
<img width="312" alt="image" src="https://user-images.githubusercontent.com/3984794/165247107-90799c67-7c56-4056-adcf-a9e4d14d7571.png">

Reply:
I see, in this case one with no padding whatsoever would likely be best! maybe call it explorer-exdoc.png or something :D